gitops/snapshot.go

```diff
@@ -169,7 +172,18 @@ func SetSnapshotIntegrationStatusAsInvalid(snapshot *applicationapiv1alpha1.Snap
 		Message: message,
 	}
 	meta.SetStatusCondition(&snapshot.Status.Conditions, condition)
-	go metrics.RegisterInvalidSnapshot(condition.Type, condition.Reason)
+	go metrics.RegisterInvalidSnapshot(AppStudioIntegrationStatusCondition, AppStudioIntegrationStatusInvalid)
+}
+
+// SetSnapshotIntegrationStatusAsError sets the AppStudio integration status condition for the Snapshot to error.
+func SetSnapshotIntegrationStatusAsError(snapshot *applicationapiv1alpha1.Snapshot, message string) {
+	condition := metav1.Condition{
+		Type:    AppStudioIntegrationStatusCondition,
+		Status:  metav1.ConditionFalse,
```

Comment:
I think this should be set to `metav1.ConditionTrue` because this status condition reports if an error occured.